ext/taglib_base/taglib_base.i

```diff
@@ -32,11 +32,12 @@ namespace TagLib {
 
 // Rename setters to Ruby convention (combining SWIG rename functions
 // does not seem to be possible, thus resort to some magic)
-%rename("%(command: ruby -e 'print(ARGV[0][3..-1].split(/(?=[A-Z])/).join(\"_\").downcase + \"=\")' )s",
+// setFoo -> foo=
+%rename("%(command: perl -e \"print lc(join('_', split(/(?=[A-Z])/, substr(@ARGV[0], 3)))), '='\" )s",
```

Comment:
Out of curiosity, why are you changing this?

Reply:
Ah, saw the commit message now. I wonder where the slowness comes from. Have you tried if adding `--no-rubygems` improves things?

I'm just a bit hesitant to introduce another dependency to building things.

Reply:
`perf stat -r 100 ruby -e 'print(ARGV[0][2..-1].split(/(?=[A-Z])/).join("_").downcase + "?")' isDone`
0,114387518 seconds time elapsed

`perf stat -r 100 ruby --disable-gems -e 'print(ARGV[0][2..-1].split(/(?=[A-Z])/).join("_").downcase + "?")' isDone`
0,021943112 seconds time elapsed

`perf stat -r 100  perl -e "print lc(join('_', split(/(?=[A-Z])/, substr(@ARGV[0], 2)))), '?'" isDone`
0,005071154 seconds time elapsed

So `perl` is 4.3 times faster than `ruby --disable-gems` which is 5.2 times faster than `ruby`. Which makes `perl` 22.5 times faster than `ruby`.

When executing `rake swig`:
- `ruby`: 3min 58s
- `ruby --disable-gems`:  3min 54s
- `perl`: 10.5s

I don't really understand why the `--disable-gems` option only shoves up a couple of seconds of the whole swig task. Based on the benchmark, I would have expected something like 45s (still seems quite long to me).

I understand your concern about introducing another dependency, but I don't think it's a big deal: Perl is part of nearly every Linux distribution out there.

Also found [that benchmark](https://github.com/chocolateboy/startup-time), who also measure a ~20x speedup from Ruby to Perl.

Reply:
Ok, seems like you have a strong opinion about this :). I'm not too fussed, and it's true that almost every system has Perl anyway. Let's go ahead with this.